frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -195,6 +198,20 @@ export const infiniteListLogic = kea<infiniteListLogicType>([
         ],
     })),
     reducers(({ props }) => ({
+        intelligentQuery: [
```

Comment:
Probably not a biggie but I initially thought there would be some AI in here

Reply:
what do we call this... `modifiedQuery` `swappedInQuery`